components/server/leeway.Dockerfile

```diff
@@ -3,33 +3,18 @@
 # See License.AGPL.txt in the project root for license information.
 
 FROM node:16.13.0-slim as builder
-
-RUN apt-get update && apt-get install -y build-essential python3
-
-COPY components-server--app /installer/
+COPY components-ws-manager-bridge--app /installer/
 
 WORKDIR /app
 RUN /installer/install.sh
 
-FROM node:16.13.0-slim
+# NodeJS v16.19
+FROM cgr.dev/chainguard/node@sha256:95bb4763acb8e9702c956e093932be97ab118db410a0619bb3fdd334c9198006
 ENV NODE_OPTIONS="--unhandled-rejections=warn --max_old_space_size=2048"
-# Using ssh-keygen for RSA keypair generation
-RUN apt-get update && apt-get install -yq \
```

Comment:
none of these seem to really be needed, and the chainguard node image doens't have `apk` anyway